pkgs/development/python-modules/qds_sdk/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib,
+ fetchFromGitHub,
+ buildPythonPackage,
+ boto,
+ inflection,
+ pytest,
+ mock,
+ requests,
+ six,
+ urllib3 }:
+
+buildPythonPackage rec {
+  pname = "qds_sdk";
+  version = "1.12.0";
+
+  # pypi does not contain tests, using github sources instead
+  src = fetchFromGitHub {
+    owner = "qubole";
+    repo = "qds-sdk-py";
+    rev = "V${version}";
+    sha256 = "18xhvlcfki8llv7fw2r5yfk20zds3gr78b4klwm9mkvhlhwds9rx";
+  };
+
+  propagatedBuildInputs = [ 
+    boto
+    inflection
+    requests
+    six 
+    urllib3 
+  ];
+
+  checkInputs = [ pytest mock ];
+  checkPhase = ''
+    py.test --disable-pytest-warnings tests
+  '';
+  doCheck = true;
```

Comment:
minor point, and doesn't really matter, but `buildPythonPackage` will default doCheck to true. This line shouldn't be needed.
```suggestion
```